src/status_im/ui/screens/activity_center/views.cljs

```diff
@@ -0,0 +1,53 @@
+(ns status-im.ui.screens.activity-center.views
+  (:require [quo.react-native :as rn]
+            [quo2.components.buttons.button :as button]
+            [quo2.components.notifications.activity-logs :as activity-logs]
+            [quo2.components.tags.context-tags :as context-tags]
+            [quo2.foundations.colors :as colors]
+            [reagent.core :as reagent]
+            [status-im.i18n.i18n :as i18n]
+            [status-im.ui.components.topbar :as topbar]
+            [status-im.utils.handlers :refer [<sub >evt]]))
+
+(defn render-notification
+  [notification index]
+  [rn/view {:flex           1
+            :flex-direction :column
+            :margin-top     (if (= 0 index) 0 4)}
+   [activity-logs/activity-log {:context   [[context-tags/group-avatar-tag "Name" {:color          :purple
+                                                                                   :override-theme :dark
+                                                                                   :size           :small
+                                                                                   :style          {:background-color colors/white-opa-10}
+                                                                                   :text-style     {:color colors/white}}]
+                                            [rn/text {:style {:color colors/white}} "did something here."]]
+                                :icon      :placeholder
+                                :message   {:body (get-in notification [:message :content :text])}
+                                :timestamp (:timestamp notification)
+                                :title     "Activity Title"
+                                :unread?   (not (:read notification))}]])
+
+(defn notifications-list
```

Comment:
not sure if it worth a separate component

Reply:
better to move to main component

Reply:
Good point, I wrote it as a separate component because I don't want the top level component of the screen (named `center`, but will be renamed to `activity-center`) to be re-processed by re-frame if only the notifications subscription changed. I believe this will become more important as the screen evolves in future PRs.

Reply:
ok